src/Core/src/Fonts/FontManager.Android.cs

```diff
@@ -143,6 +143,11 @@ public FontSize GetFontSize(Font font, float defaultFontSize = 0)
 				case "serif":
 					return Typeface.Serif;
 				default:
+					if (fontfamily.StartsWith("sansserif-", StringComparison.OrdinalIgnoreCase) ||
```

Comment:
will this cause all versions of `sanserrif-` to have the same `weight` ? 

https://android.googlesource.com/platform/frameworks/base/+/master/data/fonts/fonts.xml#117

should we set the weight based on the value after the dash?

Reply:
No the Java examples I found all pass Normal. There is no enum for Medium and Black, for example.